docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+version: '3.7'
+
+services:
+  db:
+    image: postgres:12.4
+    restart: always
+    volumes:
+    - db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: diaper
+      POSTGRES_PASSWORD: diaper
+      POSTGRES_DB: dummy
+    networks:
```

Comment:
Do you need to specify networks at all? What does this do differently than if you left it out entirely?

Reply:
I did not like the default name "diaper_default" (it doesn't say that is a network), so I specified that. However, I just noticed that docker-compose still creates the default network anyway. I just removed the network definition entirely via https://github.com/rubyforgood/diaper/pull/1893/commits/634ecf0ac7bd2731fe6448a433e5e55ec253ed5b.